pokemon_battle_arena/src/db/moves.rs

```diff
@@ -150,21 +148,21 @@ impl Technique {
                     //stats will be raised even when the target is already confused or can not be
                     //confused due to other reasons, but it will not get confused if the stats can
                     //not be raised anymore.
-                    enums::Move_Category::Swagger => {
+                    enums::MoveCategory::Swagger => {
                         if resolve::change_stats(self.get_stat_change_rate(), self.get_stat(),
                             target.clone()) {
                             resolve::ailment(self.get_name(), self.get_type(), self.get_ailment(),
                              100, target);
                         }
                     },
 
-                    enums::Move_Category::Damage_And_Lower => {
+                    enums::MoveCategory::DamageAndLower => {
                         let _ = resolve::deal_damage(self.clone(), user.clone(), target.clone());
                         let _ = resolve::change_stats(self.get_stat_change_rate(), self.get_stat(),
```

Comment:
I don't need the amount of damage that is dealt here, but in Damage and Heal it is neeeded to call the Heal function.